OpenRA.Game/Game.cs

```diff
@@ -362,10 +362,9 @@ public static void InitializeMod(string mod, Arguments args)
 
 			// Fall back to default if the mod doesn't exist
 			if (mod == null)
-				mod = args.GetValue("Engine.DefaultMod", "modchooser");
+				throw new ArgumentException("The mod to launch was not specified", "mod");
```

Comment:
How about "Game.Mod argument missing or mod could not be found"? It's a bit clearer, and gives the user the info how to fix the error.